config/puma.rb

```diff
@@ -1,3 +1,26 @@
 # frozen_string_literal: true
 
 plugin :statsd
+
+# Configure "min" to be the minimum number of threads to use to answer
+# requests and "max" the maximum.
+#
+# The default is "0, 16".
+#
+threads 32, 32
```

Comment:
Threads ~= CPU, which is what you want to optimize from the gist of the PR

This is a *lot* of threads for a Rails app. Most optimization work I've seen suggests 5-6 threads per worker. I've gone as high as 10 working stepwise over a week or two with really good memory and cpu statistics (optimizing directly on production) -- not here, but recently.

* the classic article https://www.speedshop.co/2017/10/12/appserver.html
* a more recent article: http://engineering.appfolio.com/appfolio-engineering/2019/4/11/a-simpler-rails-benchmark-puma-and-concurrency

@jbritt1 one thing to check here is that we'll need that many database connections (both redis and postgresql) in the respective pools. 64 threads seems well within normal/default configuration but its worth doublechecking because that will bring this deploy to its knees pretty fast.